### ISSUE TITLE
qutebrowser.exe detected as malware

### ISSUE THREAD

**Issue by kuspoes** [qutebrowser/qutebrowser#6093]
**Version info**:
Qutebrowser 2.0.1 latest version, I cant check using `:version` because qutebrowser.exe is detected as malware and automatically blocked by Bitdefender and Emsisoft

Windows 10 20H2

**Does the bug happen if you start with `--temp-basedir`?**:
No

**Description**
After installing v. 2.0.1, bitdefender and emsisoft marked qutebrowser.exe as a malware and quarantine it.
we uploaded the file to VirusTotal and the result was 25 antivirus / malware detected it as malware (mostly marked it as Gen: Variant.Mikey.118185). [VirusTotal result](https://www.virustotal.com/gui/file/1f1be4d41a03d059dd87583cabcd8d93b2631c117bff3dcb988f91f268c8787d/detection).

Is it safe to use?

**How to reproduce**

**Comment by fluix-dev** [comment]
Duplicate of #6081

**Comment by emanuele6** [comment]
>Does the bug happen if you start with --temp-basedir?:
No

that seems odd.

**Comment by kuspoes** [comment]
> 
> 
> > Does the bug happen if you start with --temp-basedir?:
> > No
> 
> that seems odd.

I'm sorry, it seems misinformed. 
I haven't tried it yet, because the antivirus has already deleted qutebrowser.exe

**Comment by emanuele6** [comment]
@kuspoes oh, ok. no need to try it. your configuration shouldn't affect whether the antivirus detects the binary as malware or not.

Just omit the answer to that question next time if you haven't tried it or answer with something like "Didn't try".
